voter-dapp/src/ActiveRequests.js

```diff
@@ -255,7 +255,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
         hash: computeVoteHash({
           price,
           salt,
-          account,
+          votingAccount,
```

Comment:
Just realized. This needs to be `account: votingAccount`

Reply:
Oh right, didn't realize it's an object instead of an arg list.

Reply:
Done. PTAL.

Reply:
LGTM